apps/web/app/(navigation)/MainMenu.tsx

```diff
@@ -0,0 +1,74 @@
+import * as Select from "@radix-ui/react-select";
+import {
+  SelectContentStyled,
+  SelectItemStyled,
+  SelectTriggerStyled,
+} from "./LanguageSelectorStyled";
+import styled from "styled-components";
+
+import { useHasScrollBar } from "lib/hooks/useHasScrollBar";
+import { Avatar } from "ui";
+import { useRouter } from "next/navigation";
+import { signOut, useSession } from "next-auth/react";
+import { useTranslate } from "lib/hooks";
+
+export const AvatarStyled = styled(Avatar)`
+  height: 2.1em;
+  width: 2.1em;
+`;
+
+export const MainMenu = () => {
+  const { hasScrollbar } = useHasScrollBar();
+  const router = useRouter();
+  const { data } = useSession();
+  const { t, dict } = useTranslate("MainPage");
+
+  const menuHandler = (value: string) => {
+    if (value === "sign-out") {
+      router.push("/");
+      signOut();
+    }
+    router.push("/");
+  };
```

Comment:
You don't need `router.push("/")` here, because it is handled by `next-auth` here:

https://github.com/intive/patronage2023-js/blob/5609f071eb22fb050e9fcf7ca8a1b84d67e84e6c/apps/web/pages/api/auth/%5B...nextauth%5D.ts#L83

if you also remove the `home` option then you don't need both `router.push("/")`

```suggestion
  const menuHandler = (value: string) => {
    if (value === "sign-out") {
      signOut();
    }
  };

Reply:
You right, I don't need router.push on sign-out route, but i'm leaving another one to make "Home button work"